src/index.html

```diff
@@ -1,14 +1,145 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
-  </head>
-  <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
-  </body>
+    <title>Uber</title>
+</head>
+<body>
+    <header class="main-head">
+      <div class="wrap">
+        <a href="#" class="main-head__logo"><img src="images/logo.jpg" alt="logo"></a>
+        <form class="main-head__form-wrap">
+          <label for="asap" class="main-head__label-asap">When</label>
+          <input type="text" placeholder="ASAP" id="asap" class="main-head__asap">
+          <label for="location" class="main-head__label-location">to</label>
+          <img src="images/loc.png" class="main-head__before-img" alt="location icon">
+          <input type="text" placeholder="ul. Tarasivska st, 16" id="location" class="main-head__location">
```

Comment:
если много атрибутов - делай переносы
```suggestion
          <input
             type="text"
             placeholder="ul. Tarasivska st, 16"
             id="location"
             class="main-head__location"
          >
```